src/libsyncengine/update_detection/file_system_observer/snapshot/snapshot.h

```diff
@@ -19,8 +19,8 @@
 #pragma once
 
 #include "syncpal/sharedobject.h"
-#include "snapshotitem.h"
 #include "db/dbnode.h"
+#include "snapshotitem.h"
```

Comment:
Just to avoid the change in file time line.
```suggestion
#include "snapshotitem.h"
#include "db/dbnode.h"
```

Reply:
Done